src/LodeRunner.API/src/Handlers/ResultHandler.cs

```diff
@@ -267,8 +383,20 @@ private static JsonResult CreateInternalServerErrorResponse(string message)
             return new JsonResult(new ErrorResult { Error = HttpStatusCode.InternalServerError, Message = $"Internal Server Error: {message}" })
             {
                 StatusCode = (int)HttpStatusCode.InternalServerError,
-                ContentType = JsonContentTypeApplicationJson,
+                ContentType = JsonContentTypeApplicationJsonProblem,
             };
+        }
+
+        /// <summary>
+        /// GET entity by ID.
+        /// </summary>
+        /// <typeparam name="TEntity">Model entity.</typeparam>
+        /// <param name="getResult">Gets result from data storage by ID.</param>
+        /// <param name="id">TEntity ID to search by.</param>
+        /// <returns>A task with the appropriate entity.</returns>
+        private static async Task<TEntity> GetEntityById<TEntity>(Func<string, Task<TEntity>> getResult, string id)
+        {
+            return await getResult(id);
```

Comment:
Thoughts on removing this one line function and just calling it?

Reply:
Sounds good. Removing the function.

Reply:
Removed it. Thanks